asapdiscovery/docking/scripts/run_docking_oe.py

```diff
@@ -173,7 +173,7 @@ def mp_func(out_dir, lig_name, du_name, *args, **kwargs):
         posit_methods = [""]
         chemgauss_scores = [-1.0]
         clash = -1
-        smiles = ["None"]
+        smiles = "None"
```

Comment:
Is there a special reason why we want a string here instead of the python `None` built-in?

Reply:
This is because these values on this `else` statement get written to a `.csv` on failure - I think we would prefer to have a '..,None,..' in the `.csv` rather than a '..,,..'. Would be worth revisiting if these variables get picked up in future development!